src/wrapper/jnienv.rs

```diff
@@ -325,7 +325,7 @@ impl<'a> JNIEnv<'a> {
     ///
     /// Note that resulting `JObject` can be `NULL` if `result` is `NULL`.
     pub fn pop_local_frame(&self, result: JObject) -> Result<JObject> {
-        Ok(jni_call!(self.internal, PopLocalFrame, result.into_inner()))
+        Ok(jni_non_null_call!(self.internal, PopLocalFrame, result.into_inner()).into())
```

Comment:
Shall we use `jni_non_null_call` which checks if any exceptions has occurred for a _safe_ [`PopLocalFrame`](https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/functions.html#PopLocalFrame) (= a method that does not throw exceptions, according to the spec)?